components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AbstractAuthorizationGrantHandler.java

```diff
@@ -141,6 +142,14 @@ public OAuth2AccessTokenRespDTO issue(OAuthTokenReqMessageContext tokReqMsgCtx)
         String tokenBindingReference = getTokenBindingReference(tokReqMsgCtx);
         String authorizedOrganization = getAuthorizedOrganization(tokReqMsgCtx);
 
+        // If the authorizedOrganization is deactivated or not available, return an error.
+        if (!(OAuthConstants.AuthorizedOrganization.NONE.equals(authorizedOrganization) ||
```

Comment:
Can we get "NONE" for the authorizedOrganization. It should be null or any value. If it is coming NONE, we may have to check and fix that (as it might be a mistake).

Reply:
getAuthorizedOrganization method set the authorizedOrganization to NONE if the accessing organization is null in the tokenRequestCtx
<img width="812" alt="Screenshot 2024-05-28 at 17 30 30" src="https://github.com/wso2-extensions/identity-inbound-auth-oauth/assets/25483865/c8fc1d3a-9977-4983-a29e-4643d48783d8">